ui/lib/screens/projects/projects_screen.dart

```diff
@@ -20,24 +21,31 @@ class ProjectsScreen extends StatefulWidget {
 class _ProjectsScreenState extends State<ProjectsScreen> {
   late ProjectService service;
   late Future<List<Project>> projects;
+  String _filter = '';
```

Comment:
Flutter (Dart) convention is to use `var` where possible. (The empty value allows the compiler to infer the type here.)